src/olympia/amo/cron.py

```diff
@@ -159,28 +159,6 @@ def collection_subscribers():
 
 
 @cronjobs.register
-def unconfirmed():
```

Comment:
I don't think we should be generating any users with confirmation codes and the code to support the confirmation email is now disabled since the migration was turned off a few months ago. The grace period would have happened then.

Since we disabled the migration more than 3 weeks ago I think this cron job should have already deleted these rows but there were still a lot in there (having trouble looking up the numbers but I got Wei to run the query with a `COUNT(*)` instead of the `DELETE` on prod)